src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -55,6 +67,32 @@
         <lightning:messages />
 
         <lightning:layout multipleRows="true">
+            <!--BEGIN SCOPED MATCH NOTIFICATION-->
+            <aura:if isTrue="{!or(not(empty(v.openOpportunities)), not(empty(v.unpaidPayments)))}">
+                <lightning:layoutItem size="12" smallDeviceSize="12" class="slds-p-horizontal_small">
+                    <lightning:layout class="slds-grid_align-center">
```

Comment:
@randi274 Beth and I are unsure why there's the nested lightning:layoutItem components here

Reply:
This was necessary when we needed the interior item to be smaller than the parent (we were using it to indent the div per Shannon's design), but I can remove this now that we're just letting it take the full space.